From b1c09914ee0096416ac360ee83e06639a8c0c961 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Thu, 13 Sep 2012 17:30:10 +0800
Subject: ENGR00223797-10 ARM:MX6 Kconfig:Support MX6_CLK_FOR_BOOTUI_TRANS

This patch adds CONFIG_MX6_CLK_FOR_BOOTUI_TRANS support. Setting
this Kconfig may keep enable IPU related clocks and PWM clocks
and avoid setting IPU related clocks' parents when initializing
clock tree so that bootloader splashimage can transition to
kernel smoothly.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit 21b5e4d436666967585313948a03421da1f8f68b)
---
 arch/arm/mach-mx6/Kconfig |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/mach-mx6/Kconfig b/arch/arm/mach-mx6/Kconfig
index c5d64a9..fc2190d 100644
--- a/arch/arm/mach-mx6/Kconfig
+++ b/arch/arm/mach-mx6/Kconfig
@@ -269,4 +269,13 @@ config MX6_INTER_LDO_BYPASS
 	 LDO will replaced by external pmic regulator(e.g. pfuze100), VDDCORE
 	 can be adjust automatically adjust by cpu frequency.
 
+config MX6_CLK_FOR_BOOTUI_TRANS
+	bool "MX6 clk setting for smooth UI transtion from bootloader to kernel"
+	depends on MXC_IPU_V3H
+	default n
+	help
+	 This is choosed to keep enable IPU related clocks and PWM clocks and
+	 avoid setting IPU related clocks' parents when initializing clock tree
+	 so that bootloader splashimage can transition to kernel smoothly.
+
 endif
-- 
1.7.9.5

